packages/nodes-base/nodes/Postgres/v2/transport/index.ts

```diff
@@ -65,6 +65,9 @@ export async function configurePostgres(
 		// Always return dates as ISO strings
 		[pgp.pg.types.builtins.TIMESTAMP, pgp.pg.types.builtins.TIMESTAMPTZ].forEach((type) => {
 			pgp.pg.types.setTypeParser(type, (value: string) => {
+				if (value === '-infinity' || value === 'infinity') {
+					return value;
+				}
 				return new Date(value).toISOString();
```

Comment:
This works great for `infinity` but I think Postgres supports more funky date formats that JS Dates can't parse.

For example I was able to still break it with `0099-01-08 00:00:00+00:17:30 BC`  (valid in Postgres but not JS).

Maybe it would be better to fall back to the string value when it can't be parsed as a Date? that way the node can't break from any unsupported date value. wdyt?

```suggestion
				const parsedDate = new Date(value);

				if (isNaN(parsedDate.getTime())) {
					return value;
				}

				return parsedDate.toISOString();
```

Reply:
Hi @elsmr 

That is really funny - I had added that exact check also but then removed it because I thought it was redundant, as postgres wouldn't let those values pass anyways 😅 

Thanks for the check! 👍 😊